test/widgets/docs_screenshots/ss_yandexdisk.py

```diff
@@ -0,0 +1,34 @@
+# Copyright (c) 2022 Egor Martynov
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the "Software"), to deal
+# in the Software without restriction, including without limitation the rights
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included in
+# all copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+# SOFTWARE.
+import pytest
+
+from libqtile.widget import yandexdisk
+from test.widgets.test_yandexdisk import yandexdisk_folder
+
+
+@pytest.fixture
+def widget(monkeypatch):
+    monkeypatch.setattr("libqtile.widget.yandexdisk.YandexDisk.sync_folder", yandexdisk_folder)
+    yield yandexdisk.YandexDisk
+
+
+@pytest.mark.parametrize("screenshot_manager", indirect=True)
```

Comment:
```suggestion
@pytest.mark.parametrize("screenshot_manager", [{}], indirect=True)
```
You have to pass the configuration for the widget, even if it's the default i.e. `{}`.